features/verhuisd-naar-buitenland.feature

```diff
@@ -0,0 +1,87 @@
+# language: nl
+Functionaliteit: 'verhuisd.naar-buitenland' gebeurtenis
+  Bij een aangifte van een verhuizing naar het buitenland, of naar het Caribisch deel van het Koninkrijk heeft een gebeurtenis verhuisd.naar-buitenland plaatsgevonden.
+  De gebeurtenis 'verhuisd.naar-buitenland' betekent dat de persoon verhuisd is van een Nederlandse gemeente naar het buitenland of het Caribisch deel van het Koninkrijk.
+  Wanneer geen aangifte van vertrek is gedaan, maar het vertrek uit Nederland ambthalve is geregistreerd of als gevolg van een ministrieel besluit is het niet bekend waar persoon verblijft.  
+
+  Achtergrond:
+    Gegeven het adres 'Burgemeester_Van_Der_Dussenplein_1_Hengelo'
+    * in gemeente 'Hengelo'
+    * met adresseerbaar object identificatie '0164010000047847'
+    En de persoon 'Jan'
+    * verblijft vanaf '14-4-2020' op het adres 'Burgemeester_Van_Der_Dussenplein_1_Hengelo'
+    En afnemer 'Roosendaal' is geabonneerd op 'verhuisd.emigratie' gebeurtenissen van de persoon 'Jan'
+
+  Regel: Wanneer een persoon naar het buitenland is verhuisd, dan heeft een gebeurtenis 'verhuisd.naar-buitenland' plaatsgevonden
+
+    Scenario: Aangifte van vertrek naar het buitenland
+      Gegeven het adres buitenland 'Chemin_du_Calvaire_19_Lausanne'
+      * met adresregel 1 'Chemin de Calvaire 19'
+      * met adresregel 2 'Lausanne'
+      * met adresregel 3 'Vaud'
+      * in land 'Zwitserland'
+      Als de aangifte van vertrek naar het buitenland van 'Jan' is verwerkt
+      * verblijft vanaf '1-9-2025' op het adres 'Chemin_du_Calvaire_19_Lausanne'
+      Dan is een 'verhuisd.naar-buitenland' gebeurtenis gepubliceerd met de volgende data
+      * het A-nummer van 'Jan'
+      * de datum dat 'Jan' is geemigreerd
+      * het adres 'Chemin_du_Calvaire_19_Lausanne'
+
+  Regel: Wanneer een persoon naar het buitenland is verhuisd maar nog geen definitief woonadres heeft, dan heeft een gebeurtenis 'verhuisd.naar-buitenland' plaatsgevonden.
+
+    Scenario: Aangifte van vertrek naar het buitenland en alleen het land is bekend
```

Comment:
ik zou de voorbeelden omkeren: eerst een "normaal" adres, daarna een onbekend adres (alleen land)

Reply:
Dat is toch ook zo? in het eerste scenario is toch naar een volledig bekend adres verhuisd?